src/gantt.ts

```diff
@@ -2147,19 +2316,25 @@ export class Gantt implements IVisual {
             .remove();
 
         if (this.viewModel.isParentFilled) {
-            const categoryLabelsWidth: number = this.viewModel.settings.taskLabelsCardSettings.show.value
+            let categoryLabelsWidth: number = taskLabelShow
                 ? this.viewModel.settings.taskLabelsCardSettings.width.value
-                : 0;
+                : Gantt.CollapsedAllBackgroundWidthHidden;
 
-            this.collapseAllGroup
+            categoryLabelsWidth += Gantt.CollapseAllBackgroundWidthPadding
+
+            this.collapseAllBackground = this.collapseAllGroup
                 .append("rect")
+                .classed(Gantt.CollapseAllBackground.className, true)
                 .attr("width", categoryLabelsWidth)
-                .attr("height", 2 * Gantt.TaskLabelsMarginTop)
+                .attr("x", -1)
+                .attr("y", -1)
+                .attr("height", 40); // height should be matched with axis height
```

Comment:
Can we create some variable if height should be matched with other value? So if we will change it there, it will be correct here as well. 
Also, why `-1` for x and y atrr-s?